src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,14 @@
+.catalog {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &__card {
+    margin-bottom: 48px;
+
+    &:not(:nth-child(4n)) {
+      margin-right: $gap;
```

Comment:
Also you can delete margin-bottom for your cars with 
![image](https://user-images.githubusercontent.com/33294729/173562851-d26467c5-a669-4bf4-8ab5-056d8c0d5806.png)